site/accounts/staywelcome.php

```diff
@@ -0,0 +1,11 @@
+<?php if(!defined('KIRBY')) exit ?>
+
+username: staywelcome
+password: >
+  $2a$10$LhoBWrjK9kPdmbD7/CsZVOopiTTpDRAiLCMrtucCtL5.P0O.OyIYm
```

Comment:
Also let’s please not keep this password later. This account is meant for testing.

But I agree, this needs to go to the `.gitignore`. I can do that tonight.